tree/treeplayer/test/dataframe/dataframe_friends.cxx

```diff
@@ -119,3 +119,25 @@ TEST_F(TDFAndFriends, QualifiedBranchName)
    for (auto v : t)
       EXPECT_EQ(v, 2);
 }
+
+TEST_F(TDFAndFriends, FromDefine)
+{
+   TFile f1(kFile1);
+   TTree *t1 = static_cast<TTree *>(f1.Get("t"));
```

Comment:
Why not use the type safe version.
```
TTree *t1; f1.GetObject("t",t1);
EXPECT_NE(nullptr,t1)
```
?

Reply:
Mainly because it's one line more :) In this context, we are _really really sure_ that the
tree exists anyway. It would be nice to have `f.Get<TTree>("t")` by the way..